src/dxtbx/sequence_filenames.py

```diff
@@ -181,6 +183,9 @@ def replace_template_format_with_hash(match):
 
 def template_string_to_glob_expr(template):
     """Convert the template to a glob expression."""
+    if template.count("#") == 1:
```

Comment:
We should probably have a test for this?

Reply:
I appreciate it is a null op, though (actually shouldn't this replace with e.g. `[0-9]*` or whatever it is to allow an arbitrary number of digits?

Reply:
Agree it seems this is not tested. I'll add something. A test of `template_image_range` would do, as that should exercise all the added code